cmd/virt-launcher/Dockerfile

```diff
@@ -23,21 +23,15 @@ MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 RUN dnf -y install \
   socat \
   genisoimage \
-  util-linux \
-  libcgroup-tools \
-  ethtool \
-  net-tools \
-  sudo && dnf -y clean all && \
+  && dnf -y clean all && \
```

Comment:
I wonder if there's any trick here that we can use to make it work with yum-based distros as well with no changes. cc @sreichar 
Not for this specific PR but a thought in general.

Reply:
We can easily get rid of `socat`, also `qemu-img` is part of the base libvirt image. My preferred solution is to also get rid of `genisoimage` too and don't install anyting extra at all. Need to investigate how hard it is to get rid of `genisoimage`.

Reply:
And you can also install things using pkcon ok distros that have PackageKit installed. I'm not sure how many base docker images have that, though.